samples/SkiaMultiplatformSample/src/commonMain/kotlin/org/jetbrains/skiko/sample/Clocks.kt

```diff
@@ -94,12 +89,23 @@ class Clocks: SkikoView {
     }
 
     override fun onPointerEvent(event: SkikoPointerEvent) {
-        if (event.isLeftClick) {
-            xpos = event.x
-            ypos = event.y
+        when (event.kind) {
+            SkikoPointerEventKind.MOVE -> {
+                xpos = event.x
+                ypos = event.y
+            }
+            else -> { println("PointerEvent: ${event.modifiers} + ${event.kind} + ${event.buttons}") }
         }
     }
 
+    override fun onInputEvent(event: SkikoInputEvent) {
+        // TODO: provide examplec
+    }
+
+    override fun onKeyboardEvent(event: SkikoKeyboardEvent) {
+        println("KeyboardEvent: ${event.kind} ${event.modifiers} + ${event.code}")
```

Comment:
unneeded print

Reply:
I've removed all event output for now - I have thoughts on a better future example.